packages/katello/katello/hostname-change.rb

```diff
@@ -345,6 +345,53 @@ def run
       STDOUT.puts "updating hostname in foreman installer scenarios"
       self.run_cmd("sed -i -e 's/#{@old_hostname}/#{@new_hostname}/g' #{scenarios_path}/*.yaml")
 
+      if @scenario_answers['foreman_proxy']['dns']
+        STDOUT.puts 'gathering DNS data...'
+        local_ip = '127.0.0.1'
+        zone = @scenario_answers['foreman_proxy']['dns_zone']
+        old_fqdn = @old_hostname
```

Comment:
why reassign @old_hostname?

Reply:
@johnpmitsch  It's just for readability, because I wanted to be very clear about what values I'm giving to nsupdate.  To me "hostname" could mean either the FQDN (a.b.c.d) or just the basic hostname (a), so calling it "fqdn" removes the ambiguity.